<?xml version="1.0"?>
<launch>  
<arg name="read_fast" default="false" />
<arg name="read_once" default="false" />
<arg name="repeat_delay" default="0.0" />
<param name="model" value="R-Fans-32" /> <!-- R-Fans-32 R-Fans-16 C-Fans-128  C-Fans-32 -->
<node pkg="rfans_driver" type="driver_node" name="rfans_driver" output="screen">
  <param name="advertise_name" value="rfans_packets" />
  <param name="control_name" value="rfans_control"/>
  <param name="device_ip" value="192.168.0.3" />
  <param name="device_port" value="2014" />
  <param name="rps" value="10"/>
  <param name="readfile_path"  value=""/>
  <!-- <param name="readfile_path"  value="/home/bkth/qt_ws/src/StarROS/usr/date/Cfan32/data-20200102-142559-29.000.isf"/>-->
  <param name="cfg_path"  value=""/>
  <!-- <param name="cfg_path"  value="/home/bkth/qt_ws/src/StarROS/usr/date/Cfan32/revise.ini"/>-->
  <param name="save_xyz"  value="false"/>
  <param name="OutExport_path"  value=""/>
  <!-- <param name="OutExport_path"  value="/home/bkth/qt_ws/src/StarROS/usr/date"/>-->
  <param name="use_double_echo" value="false"/><!-- Whether to use use_double_echo -->
  <param name="use_gps" value="false"/>
  <param name="read_fast" value="$(arg read_fast)" />
  <param name="read_once" value="$(arg read_once)" />
  <param name="repeat_delay" value="$(arg repeat_delay)" />
  <param name ="cut_angle_range" value="360.0"/>
</node>

</launch>
